src/main/kotlin/build/buf/gradle/BufYamlGenerator.kt

```diff
@@ -0,0 +1,47 @@
+package build.buf.gradle
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.gradle.api.GradleException
+import java.io.File
+
+class BufYamlGenerator {
```

Comment:
Let's make this `internal` so nobody can depend on it.

Reply:
Done